man/ibd2sdi.1

```diff
@@ -2,24 +2,12 @@
 .\"     Title: ibd2sdi
 .\"    Author: [FIXME: author] [see http://docbook.sf.net/el/author]
 .\" Generator: DocBook XSL Stylesheets v1.79.1 <http://docbook.sf.net/>
-<<<<<<< HEAD
-.\"      Date: 06/02/2023
-||||||| b5da0b9817c
-.\"      Date: 03/03/2023
-=======
 .\"      Date: 06/20/2023
->>>>>>> mysql-8.1.0
 .\"    Manual: MySQL Database System
 .\"    Source: MySQL 8.1
 .\"  Language: English
 .\"
-<<<<<<< HEAD
-.TH "IBD2SDI" "1" "06/02/2023" "MySQL 8\&.0" "MySQL Database System"
-||||||| b5da0b9817c
-.TH "IBD2SDI" "1" "03/03/2023" "MySQL 8\&.0" "MySQL Database System"
-=======
 .TH "IBD2SDI" "1" "06/20/2023" "MySQL 8\&.1" "MySQL Database System"
```

Comment:
My understanding of this is because upstream based their 8.1 branch in some commit between 8.0.33 and 8.0.34, to be more precise b5da0b9817c552f0ec30724529df2e912404947e is their common ancestor. From this point forward they were two independent branches.
8.0.34 branch changed those man files updating it to their data.
8.1 also changed those man files, but on a different commit

Our 8.1 was based on our 8.0.34 which had the 8.0.34 changes on those files, and when trying to merge the 8.1 changes on the same file, we got the conflicts.
This will be an issue only on first merge. It was simpler to base on our 8.1 with our last version (8.0.34) than having to rewind 8.0.33 and partially merge up to GCA point and then merge again 8.1

Reply:
Thanks for the explanation. If it is one-time only, then we are all good.